crypto/fipsmodule/rsa/internal.h

```diff
@@ -151,6 +151,14 @@ int rsa_sign_no_self_test(int hash_nid, const uint8_t *digest,
                           size_t digest_len, uint8_t *out, unsigned *out_len,
                           RSA *rsa);
 
+int rsa_digestsign_no_self_test(const EVP_MD *md, const uint8_t *input,
```

Comment:
I suggest adding comments here similar to those in ecdsa/internal.h.

Reply:
Sure, I initially didn't add them since the section on L133~150 encapsulated all of these, but I'll add more comments for additional clarification.